### ISSUE TITLE
Incompatibility of ilp between versions 1.3.2 and 1.3.3

### ISSUE THREAD

**Issue by pwalczysko** [ilastik/ilastik#2230]
Dear Ilastik team

## Describe the bug
I have a Pixel classification Project created in Ilastik 1.3.2, in summer 2019. This project successfully works in Ilastik 1.3.2 and is perfectly functional. It is also used to run Ilastik 1.3.2 headless. Its name is ``pixel-class-wednesday.ilp`` It does a pixel classification on z-stacks on images such as http://idr.openmicroscopy.org/webclient/?show=image-6001240
When I try to open the ``pixel-class-wednesday.ilp`` in Ilastik 1.3.3, I get following error message. (see screenshot1 below). 
Note that the Ilastik 1.3.3 and Ilastik 1.3.2 are running on the same machine (Mac OS High Sierra), and the paths to raw data as saved in the project are valid.
Treid to attach the ``pixel-class-wednesday.ilp`` here too, but github does not support attaching of such files. Happy to give you the file by other means if you tell me how.

![Screen Shot 2020-04-03 at 10 27 02](https://user-images.githubusercontent.com/2478303/78352625-8aac8280-75a0-11ea-9785-b17fb52bbb51.png)

## Expected behavior
Ilastik 1.3.3 should be opening the ``pixel-class-wednesday.ilp`` exactly as Ilastik 1.3.2 does. There is no reason not to - the environment is the same, all that changed is the Ilastik version. A minor version change should not break the compatibility.

## To Reproduce
1. Open the attached ``pixel-class-wednesday.ilp`` in Ilastik 1.3.3.
2. Observe the error.

## Error message/traceback

I am adding what I found in ``Ilastik_log.txt`` in a collapsible section with markdown below
<details>
  <summary>Click to expand!</summary>

```
INFO 2020-04-03 11:50:11,817 ilastik_main 4825 140736069866368 Starting ilastik from "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release".
INFO 2020-04-03 11:50:12,526 filterOperators 4825 140736069866368 Using fast filters.
WARNING 2020-04-03 11:50:12,530 warnings 4825 140736069866368 __init__.py(20): UserWarning: init: Could not import tiktorch classifier
WARNING 2020-04-03 11:50:14,234 opConservationTracking 4825 140736069866368 Could not find any ILP solver
WARNING 2020-04-03 11:50:14,245 opStructuredTracking 4825 140736069866368 Could not find any ILP solver
WARNING 2020-04-03 11:50:14,248 structuredTrackingWorkflow 4825 140736069866368 Could not find any learning solver. Tracking will use flow-based solver (DPCT). Learning for tracking will be disabled!
INFO 2020-04-03 11:51:28,539 projectManager 4825 140736069866368 Opening Project: /Users/pwalczysko/Downloads/pixel-class-wednesday.ilp
WARNING 2020-04-03 11:51:28,543 warnings 4825 140736069866368 dataset.py(313): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
INFO 2020-04-03 11:51:28,701 acceleratesupport 4825 140736069866368 No OpenGL_accelerate module loaded: No module named 'OpenGL_accelerate'
WARNING 2020-04-03 11:51:28,896 warnings 4825 140736069866368 slot.py(1287): UserWarning: Deprecated use slot.operator property instead
WARNING 2020-04-03 11:51:29,221 warnings 4825 140736069866368 opSimpleBlockedArrayCache.py(43): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING 2020-04-03 11:51:29,229 warnings 4825 140736069866368 opSlicedBlockedArrayCache.py(141): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
ERROR 2020-04-03 11:51:29,474 log_exception 4825 140736069866368 Traceback (most recent call last):
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/shell/projectManager.py", line 460, in _loadProject
    serializer.deserializeFromHdf5(self.currentProjectFile, projectFilePath, self._headless)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/base/appletSerializer.py", line 1151, in deserializeFromHdf5
    ss.deserialize(topGroup)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/pixelClassification/pixelClassificationSerializer.py", line 100, in deserialize
    super().deserialize(group)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/base/appletSerializer.py", line 264, in deserialize
    self._deserialize(group[self.name], self.inslot)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/lazyflow/lazyflow/utility/timer.py", line 153, in wrapper
    return func(*args, **kwargs)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/base/appletSerializer.py", line 620, in _deserialize
    blockArray, slicing, self.inslot[index], Project(mygroup.file)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/pixelClassification/pixelClassificationSerializer.py", line 82, in reshape_datablock_and_slicing_for_input
    fixed_block = Array5D(block, saved_data_axiskeys).raw(current_axiskeys)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/Resources/lib/python3.7/site-packages/ndstructs/array5D.py", line 84, in __init__
    assert len(arr.shape) == len(axiskeys)
AssertionError

ERROR 2020-04-03 11:51:29,475 log_exception 4825 140736069866368 Project could not be loaded due to the exception shown above.
Aborting Project Open Action
ERROR 2020-04-03 11:51:29,476 log_exception 4825 140736069866368 Traceback (most recent call last):
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/shell/gui/ilastikShell.py", line 1542, in _loadProject
    self.projectManager._loadProject(hdf5File, projectFilePath, readOnly)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/lazyflow/lazyflow/utility/timer.py", line 153, in wrapper
    return func(*args, **kwargs)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/shell/projectManager.py", line 460, in _loadProject
    serializer.deserializeFromHdf5(self.currentProjectFile, projectFilePath, self._headless)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/base/appletSerializer.py", line 1151, in deserializeFromHdf5
    ss.deserialize(topGroup)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/pixelClassification/pixelClassificationSerializer.py", line 100, in deserialize
    super().deserialize(group)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/base/appletSerializer.py", line 264, in deserialize
    self._deserialize(group[self.name], self.inslot)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/lazyflow/lazyflow/utility/timer.py", line 153, in wrapper
    return func(*args, **kwargs)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/base/appletSerializer.py", line 620, in _deserialize
    blockArray, slicing, self.inslot[index], Project(mygroup.file)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/pixelClassification/pixelClassificationSerializer.py", line 82, in reshape_datablock_and_slicing_for_input
    fixed_block = Array5D(block, saved_data_axiskeys).raw(current_axiskeys)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/Resources/lib/python3.7/site-packages/ndstructs/array5D.py", line 84, in __init__
    assert len(arr.shape) == len(axiskeys)
AssertionError
```

</details>

## Screenshots
If applicable, please add screenshots to help explain your problem.
![Screen Shot 2020-04-03 at 10 27 02](https://user-images.githubusercontent.com/2478303/78353677-60f45b00-75a2-11ea-92f2-d05d6174f172.png)

## Desktop (please complete the following information):
 - ilastik version: 1.3.3post2-OSX
 - OS: [macOS 10.13.6: High Sierra
 
All the best

Petr

**Comment by k-dominik** [comment]
Hi @pwalczysko,

thank you very much for the report. Backwards compatibility is important to us, so we are completely with you with your expectation that loading the project should just work. I tried to quickly reproduce it and created a pixel classification project with 1.3.2 and had no problems opening it in 1.3.3.

So, yes, if you wouldn't mind sharing your project with us, could you please upload it to [this filedrop](https://oc.embl.de/index.php/s/5yQqvrJZyCM5XrK).

In the meantime there are two things you can try on your side:

1) Download the [latest beta (`1.4.1b1`)](https://www.ilastik.org/download.html#beta) and check if it works there.
1) Try to load the project by going through the import: _Project -> Import Project_
     You will first be asked for the project you want to import, then for the name and location of the new project file and finally to which workflow type the project should be imported (Pixel Classification in your case)

**Comment by pwalczysko** [comment]
Hi @k-dominik 
thanks for the quick answer. I just uploaded the file to the indicated place.
I will have a look at the other two workflows you are suggesting.

**Comment by pwalczysko** [comment]
Hi @k-dominik 

Tried to do the ``Project -> Import Project`` workflow suggested by you in Ad 2 above.
I have the same experience, exactly the same error popup (after I have selected the ``pixel-class-wednesday.ilp``, re-saved it and adjusted the module as instructed. 

From the ilastik.log (this also on the first look seems quite similar, but see yourself below)

<details>
  <summary>Click to expand!</summary>


```
INFO 2020-04-03 13:17:28,868 ilastik_main 7018 140736069866368 Starting ilastik from "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release".
INFO 2020-04-03 13:17:29,563 filterOperators 7018 140736069866368 Using fast filters.
WARNING 2020-04-03 13:17:29,567 warnings 7018 140736069866368 __init__.py(20): UserWarning: init: Could not import tiktorch classifier
WARNING 2020-04-03 13:17:31,284 opConservationTracking 7018 140736069866368 Could not find any ILP solver
WARNING 2020-04-03 13:17:31,293 opStructuredTracking 7018 140736069866368 Could not find any ILP solver
WARNING 2020-04-03 13:17:31,296 structuredTrackingWorkflow 7018 140736069866368 Could not find any learning solver. Tracking will use flow-based solver (DPCT). Learning for tracking will be disabled!
INFO 2020-04-03 13:17:52,503 acceleratesupport 7018 140736069866368 No OpenGL_accelerate module loaded: No module named 'OpenGL_accelerate'
WARNING 2020-04-03 13:17:52,749 warnings 7018 140736069866368 slot.py(1287): UserWarning: Deprecated use slot.operator property instead
WARNING 2020-04-03 13:17:52,938 warnings 7018 140736069866368 dataset.py(313): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING 2020-04-03 13:17:53,082 warnings 7018 140736069866368 opSimpleBlockedArrayCache.py(43): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING 2020-04-03 13:17:53,090 warnings 7018 140736069866368 opSlicedBlockedArrayCache.py(141): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
ERROR 2020-04-03 13:17:53,264 log_exception 7018 140736069866368 Traceback (most recent call last):
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/shell/gui/ilastikShell.py", line 1545, in _loadProject
    self.projectManager._importProject(importFromPath, hdf5File, projectFilePath)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/shell/projectManager.py", line 519, in _importProject
    serializer.deserializeFromHdf5(self.currentProjectFile, newProjectFilePath, self._headless)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/base/appletSerializer.py", line 1151, in deserializeFromHdf5
    ss.deserialize(topGroup)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/pixelClassification/pixelClassificationSerializer.py", line 100, in deserialize
    super().deserialize(group)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/base/appletSerializer.py", line 264, in deserialize
    self._deserialize(group[self.name], self.inslot)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/lazyflow/lazyflow/utility/timer.py", line 153, in wrapper
    return func(*args, **kwargs)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/base/appletSerializer.py", line 620, in _deserialize
    blockArray, slicing, self.inslot[index], Project(mygroup.file)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/pixelClassification/pixelClassificationSerializer.py", line 82, in reshape_datablock_and_slicing_for_input
    fixed_block = Array5D(block, saved_data_axiskeys).raw(current_axiskeys)
  File "/Applications/ilastik-1.3.3post2-OSX.app/Contents/Resources/lib/python3.7/site-packages/ndstructs/array5D.py", line 84, in __init__
    assert len(arr.shape) == len(axiskeys)
AssertionError
```

</details>

**Comment by pwalczysko** [comment]
And with the latest beta version of Ilastik, I am getting the same experience.
This is your suggestion Ad 1. above.
Again, the error from ilastik log see below please.

Expand details to see the error
<details>

```
INFO 2020-04-03 13:37:44,185 ilastik_main 7528 140736069866368 Starting ilastik from "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release".
INFO 2020-04-03 13:37:44,868 filterOperators 7528 140736069866368 Using fast filters.
WARNING 2020-04-03 13:37:46,598 opConservationTracking 7528 140736069866368 Could not find any ILP solver
WARNING 2020-04-03 13:37:46,609 opStructuredTracking 7528 140736069866368 Could not find any ILP solver
WARNING 2020-04-03 13:37:46,613 structuredTrackingWorkflow 7528 140736069866368 Could not find any learning solver. Tracking will use flow-based solver (DPCT). Learning for tracking will be disabled!
INFO 2020-04-03 13:38:09,544 projectManager 7528 140736069866368 Opening Project: /Users/pwalczysko/Downloads/pixel-class-wednesday.ilp
WARNING 2020-04-03 13:38:09,545 warnings 7528 140736069866368 projectManager.py(158): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING 2020-04-03 13:38:09,547 warnings 7528 140736069866368 projectManager.py(168): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING 2020-04-03 13:38:09,563 pixelClassificationWorkflow 7528 140736069866368 Unused command-line args: ['-psn_0_6170082']
INFO 2020-04-03 13:38:09,655 acceleratesupport 7528 140736069866368 No OpenGL_accelerate module loaded: No module named 'OpenGL_accelerate'
WARNING 2020-04-03 13:38:09,914 warnings 7528 140736069866368 slot.py(1290): UserWarning: Deprecated use slot.operator property instead
WARNING 2020-04-03 13:38:10,160 warnings 7528 140736069866368 dataSelectionSerializer.py(385): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING 2020-04-03 13:38:10,162 warnings 7528 140736069866368 featureSelectionSerializer.py(91): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING 2020-04-03 13:38:10,163 warnings 7528 140736069866368 featureSelectionSerializer.py(96): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING 2020-04-03 13:38:10,165 warnings 7528 140736069866368 featureSelectionSerializer.py(102): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING 2020-04-03 13:38:10,165 warnings 7528 140736069866368 featureSelectionSerializer.py(122): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
WARNING 2020-04-03 13:38:10,253 warnings 7528 140736069866368 opSimpleBlockedArrayCache.py(43): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING 2020-04-03 13:38:10,262 warnings 7528 140736069866368 opSlicedBlockedArrayCache.py(141): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING 2020-04-03 13:38:10,267 warnings 7528 140736069866368 featureSelectionSerializer.py(169): H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
ERROR 2020-04-03 13:38:10,434 log_exception 7528 140736069866368 Traceback (most recent call last):
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/shell/projectManager.py", line 460, in _loadProject
    serializer.deserializeFromHdf5(self.currentProjectFile, projectFilePath, self._headless)
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/base/appletSerializer.py", line 1151, in deserializeFromHdf5
    ss.deserialize(topGroup)
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/pixelClassification/pixelClassificationSerializer.py", line 100, in deserialize
    super().deserialize(group)
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/base/appletSerializer.py", line 264, in deserialize
    self._deserialize(group[self.name], self.inslot)
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release/ilastik-meta/lazyflow/lazyflow/utility/timer.py", line 153, in wrapper
    return func(*args, **kwargs)
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/base/appletSerializer.py", line 620, in _deserialize
    blockArray, slicing, self.inslot[index], Project(mygroup.file)
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/pixelClassification/pixelClassificationSerializer.py", line 82, in reshape_datablock_and_slicing_for_input
    fixed_block = Array5D(block, saved_data_axiskeys).raw(current_axiskeys)
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/Resources/lib/python3.7/site-packages/ndstructs/array5D.py", line 84, in __init__
    assert len(arr.shape) == len(axiskeys)
AssertionError

ERROR 2020-04-03 13:38:10,434 log_exception 7528 140736069866368 Project could not be loaded due to the exception shown above.
Aborting Project Open Action
ERROR 2020-04-03 13:38:10,435 log_exception 7528 140736069866368 Traceback (most recent call last):
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/shell/gui/ilastikShell.py", line 1542, in _loadProject
    self.projectManager._loadProject(hdf5File, projectFilePath, readOnly)
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release/ilastik-meta/lazyflow/lazyflow/utility/timer.py", line 153, in wrapper
    return func(*args, **kwargs)
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/shell/projectManager.py", line 460, in _loadProject
    serializer.deserializeFromHdf5(self.currentProjectFile, projectFilePath, self._headless)
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/base/appletSerializer.py", line 1151, in deserializeFromHdf5
    ss.deserialize(topGroup)
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/pixelClassification/pixelClassificationSerializer.py", line 100, in deserialize
    super().deserialize(group)
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/base/appletSerializer.py", line 264, in deserialize
    self._deserialize(group[self.name], self.inslot)
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release/ilastik-meta/lazyflow/lazyflow/utility/timer.py", line 153, in wrapper
    return func(*args, **kwargs)
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/base/appletSerializer.py", line 620, in _deserialize
    blockArray, slicing, self.inslot[index], Project(mygroup.file)
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/ilastik-release/ilastik-meta/ilastik/ilastik/applets/pixelClassification/pixelClassificationSerializer.py", line 82, in reshape_datablock_and_slicing_for_input
    fixed_block = Array5D(block, saved_data_axiskeys).raw(current_axiskeys)
  File "/Applications/ilastik-1.4.0b1-OSX.app/Contents/Resources/lib/python3.7/site-packages/ndstructs/array5D.py", line 84, in __init__
    assert len(arr.shape) == len(axiskeys)
AssertionError

```

</details>

Btw, I actually am fine atm with what I need to do. I just re-created the ilp file again from scratch (using the raw data and Ilastik 1.3.3). 
Interestingly, this new ilp (pixel-class-133.ilp) is not opening in the 1.3.2 Ilastik, albeit with a different error popup.

![Screen Shot 2020-04-03 at 11 15 36](https://user-images.githubusercontent.com/2478303/78361868-01517c00-75b1-11ea-934a-5201769de023.png)

**Comment by k-dominik** [comment]
Hi @pwalczysko,

I can reproduce it just fine now with your project file. That really sucks.

for the time being, since you're recreating the project, you can at least use the same labels! Load the project in 1.3.2, go to the training applet and, right click on the Labels layer and choose export. Export your labels (ideally to an h5 file) and close the project. In the new version you can then go to the training applet and say "import layers"... the procedure is outline [in our docs](https://www.ilastik.org/documentation/pixelclassification/pixelclassification#import)




> Interestingly, this new ilp (pixel-class-133.ilp) is not opening in the 1.3.2 Ilastik, albeit with a different error popup.

well, yeah. In general we would like to maintain forward compatibility (so loading a project in older versions) but this is, quite frankly, sometimes too big of a burden :/

we'll look into the original issue of course!

**Comment by k-dominik** [comment]
this issue does not occur with current master

**Comment by joshmoore** [comment]
@k-dominik, could you summarize what compabilities you expect to be supported? e.g. was 1.3.3 a one off, and 1.3.2 and master are compatible?
